hapi-fees/src/main/java/com/hedera/services/pricing/RequiredPriceTypes.java

```diff
@@ -0,0 +1,77 @@
+package com.hedera.services.pricing;
+
+/*-
+ * ‌
+ * Hedera Services Node
```

Comment:
should be `Hedera Services API Fees` for this file and all files in `hapi-fees`. I hope it is not generated from `mvn license:update-file-header` or we have to investigate and fix it.